integration_tests/tests/consistency/consistency_account_report.sql

```diff
@@ -0,0 +1,45 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with prod as (
+    select
+        account_id,
+        sum(clicks) as clicks, 
+        sum(impressions) as impressions,
+        sum(spend) as spend
```

Comment:
Not a requirement before release, but should we consider including the conversion fields in here as well (commented out) so we don't forget to include them in subsequent releases?

Reply:
Same comment for other consistency tests

Reply:
added